01-landing-page/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="scss/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <title>Queeg</title>
+</head>
+
+<body>
+    <!-- Navbar -->
+    <nav class='nav'>
+        <div class="container">
+            <div class="d-flex">
+                <div class="nav__logo">
+                    <a class='logo__link'href="#">Queeg<span class='trademark'>&trade;</span></a>
+                </div>
+                <div class="nav__list">
+                    <ul class='d-flex'>
+                        <li class='nav__list__item'>
+                            <a class='nav__link' href="">Product</a>
+                        </li>
+                        <li class='nav__list__item'>
+                            <a class='nav__link' href="">Business</a>
+                        </li>
+                        <li class='nav__list__item'>
+                            <a class='nav__link' href="">About</a>
+                        </li>
+                        <li class='nav__list__item'>
+                            <a class='nav__link' href="">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="menu__toggler">
+                    <div class="menu__toggler__icon"></div>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <!-- Hero -->
+    <section class="hero container">
+        <div class="hero__copy">
+            <h1 class='hero__heading'>Embrace <br> Blockchain<br> with Condifence</h1>
```

Comment:
`<br>` should not be used for visual line breaks. It should only be used where it's purpose has a meaning, for example a poem or address. Here the best is using `max-width` so the text can be better responsive